src/App.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from 'react';
+
 import './App.css';
-import Header from './Component/header';
+import Landing from './pages/landing';
```

Comment:
Names in the same way as the classes. In this instance, the file name should start with a capital letter.